packages/kbn-search-connectors/lib/collect_connector_stats.ts

```diff
@@ -113,6 +113,7 @@ export const collectConnectorStats = async (
     const connectorStats: ConnectorStats = {
       id: connectorId,
       isDeleted: true,
+      serviceType: orphanedSyncJobs[0].connector.service_type,
```

Comment:
This assumes there's an `orphanedSyncJobs`, but that can be undefined. Should be: 
```
orphanedSyncJobs[0]?.connector.service_type
```

Reply:
The `orphanedSyncJobs` is always non-empty because it's from `syncJobsMap` and this is how it's built:
https://github.com/elastic/kibana/blob/b822c6fbd09762cfb89d19507174cd6733f001af/packages/kbn-search-connectors/lib/collect_connector_stats.ts#L46